pkg/slices/slices_test.go

```diff
@@ -0,0 +1,297 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package slices
+
+import (
+	"reflect"
+	"strings"
+	"testing"
+)
+
+func TestEqual(t *testing.T) {
+	tests := []struct {
+		name      string
+		elements  []string
+		elements2 []string
+		want      bool
+	}{
+		{
+			name:      "both empty",
+			elements:  []string{},
+			elements2: []string{},
+			want:      true,
+		},
+		{
+			name:      "first empty",
+			elements:  []string{},
+			elements2: []string{"a", "b"},
+			want:      false,
+		},
+		{
+			name:      "first contains all second",
+			elements:  []string{"a", "b", "c"},
+			elements2: []string{"a", "b"},
+			want:      false,
+		},
+		{
+			name:      "equal",
+			elements:  []string{"a", "b"},
+			elements2: []string{"a", "b"},
+			want:      true,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := Equal(tt.elements, tt.elements2); got != tt.want {
```

Comment:
this is just an alias for the upstream package, I am not sure we need to test it?

Reply:
> this is just an alias for the upstream package, I am not sure we need to test it?

I removed the test method for the alias function